src/fluree/db/ledger/transact/validation.clj

```diff
@@ -566,8 +566,10 @@
               result-ch (async/chan parallelism)
               af        (fn [f res-chan]
                           (async/go
-                            (let [fn-result (as-> (f tx-state*) res
-                                                  (if (channel? res) (async/<! res) res))]
+                            (let [fn-result (try
+                                              (as-> (f tx-state*) res
+                                                    (if (channel? res) (async/<! res) res))
+                                              (catch Exception e e))]
```

Comment:
This is what `go-try` and `<?` are intended to help with. Would that work here? Something like (starting on line 568):

```clojure
(go-try
  (let [fn-result (f tx-state*)]
    (if (channel? fn-result)
      (<? fn-result)
      fn-result)))
```